paperqa/utils.py

```diff
@@ -345,6 +346,24 @@ def remove_substrings(target: str, substr_removal_list: Collection[str]) -> str:
     return target
 
 
+def convert_acutes(text: str) -> str:
```

Comment:
A naming nit is `convert_acutes` is not a clear name imo, I don't think "acutes" is a well known term. Maybe go to `replace_acute_accents`